frontend/src/metabase/query_builder/components/view/DataReferenceButton/DataReferenceButton.tsx

```diff
@@ -1,22 +1,29 @@
-/* eslint-disable react/prop-types */
 import { t } from "ttag";
+import cx from "classnames";
+
 import { Icon } from "metabase/core/components/Icon";
 import Tooltip from "metabase/core/components/Tooltip";
-import NativeQuery from "metabase-lib/queries/NativeQuery";
-import { ButtonRoot } from "./DataReferenceButton.styled";
+
+interface DataReferenceButtonProps {
+  className?: string;
+  isShowingDataReference: boolean;
+  size: number;
+  toggleDataReference: () => void;
+}
 
 export const DataReferenceButton = ({
+  className,
   isShowingDataReference,
-  toggleDataReference,
   size,
-  className,
-}) => (
+  toggleDataReference,
+}: DataReferenceButtonProps) => (
   <Tooltip tooltip={t`Learn about your data`}>
-    <ButtonRoot className={className} isSelected={isShowingDataReference}>
+    <a
```

Comment:
What's the reason we can't use `ButtonRoot`?

Also, if we don't want to really use it we should also remove `frontend/src/metabase/query_builder/components/view/DataReferenceButton/DataReferenceButton.styled.tsx` since there is only `ButtonRoot` defined there, and we won't use it anymore.

Reply:
🤦 Thanks, Kelvin, that's a silly mistake. TDLR it's a "merge" conflict — I had TypeScript conversion changes on a local branch before #32526, then Alex's PR extracted styled-components, and I had to solve a few conflicts. I believe here I didn't pick the right diff. Thanks for being on the lookout as always 🙏